src/server/app/core/middlewares/version-middleware.ts

```diff
@@ -0,0 +1,8 @@
+import { NextFunction, Request, Response } from 'express';
+
+const version = require('../../../../../package.json').version;
```

Comment:
I think it is safely to use process.cwd() instead since it is always at the root